charts/prometheus-postgres-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.8.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 1.4.0
+version: 1.4.1
```

Comment:
new feature should rais minor version to 1.5.0

Reply:
I fix it